### PR TITLE
feat: test network long blocks

### PR DESCRIPTION
**Pull request by atanmarko**
# Description

Added option for custom block time when starting Polygon Edge, `BLOCK_TIME`. Default is 2 secods.

# Changes include

- [x] New feature (non-breaking change that adds functionality)

# Checklist

- [x] I have assigned this PR to myself
- [x] I have added at least 1 reviewer
- [ ] I have added the relevant labels
- [ ] I have updated the official documentation
- [ ] I have added sufficient documentation in code

## Testing

- [] I have tested this code with the official test suite
- [x] I have tested this code manually

### CODE REVIEW

**Comment by sebastiendan** [comment]
@atanmarko Why don't we use an env var to dynamically set the block time as we do with other fields, then we don't need an extra init.sh command and we can still set 2s as the default block time.

File: consensus/ibft/signer/helper_test.go [hunk 1]
```diff
@@ -73,7 +73,7 @@ func Test_wrapCommitHash(t *testing.T) {
 	assert.Equal(t, expectedOutput, output)
 }
 
-// nolint
+//nolint
```
Comment (sebastiendan) [inline consensus/ibft/signer/helper_test.go:76]: Maybe revert this unnecessary change?

**Review by sebastiendan — APPROVED** [review]

File: consensus/ibft/signer/helper_test.go (hunk 1 above)
Comment (atanmarko) [reply consensus/ibft/signer/helper_test.go:76]: It is not unnecessary,CI fails because of that, check [here](https://github.com/topos-protocol/polygon-edge/actions/runs/6197355135/job/16825733802)

**Review by gruberb — APPROVED** [review]
👍
